build_tools/kokoro/gcp_ubuntu/simulate_kokoro.sh

```diff
@@ -15,14 +15,15 @@
 # limitations under the License.
 
 # Simulates the behavior of Kokoro on a local machine.
-# Usage:
+# Usage (under sudo):
```

Comment:
This isn't necessarily true. Running docker requires sudo or being in the docker group. I think we have a PR out for updating the documentation in our docker readme to talk about this weirdness, but looks like it got lost: https://github.com/google/iree/pull/5088

Reply:
Yeah I had it on sudo but it got really annoying because it meant all the files it created were root. Docker is kind of djanky :stuck_out_tongue: